.github/workflows/builder.yml

```diff
@@ -33,10 +33,10 @@ on:
       - setup.py
 
 env:
-  DEFAULT_PYTHON: 3.9
+  DEFAULT_PYTHON: "3.10"
   BUILD_NAME: supervisor
   BUILD_TYPE: supervisor
-  WHEELS_TAG: 3.9-alpine3.14
+  WHEELS_TAG: musllinux_1_2
```

Comment:
It changed how it work, we don't need that env variable or make it configurable anymore

Reply:
Gonna let the CI run to make sure everything else passes then make this.